Fixed multiple B.3.16.* features

mmcev106 opened the pull request:
@kcmcg, I'm happy to talk through anything if it's unclear or you have concerns.

kcmcg commented:
Is this PR the driving reason why the csv date file has been updated, or was there a need to update this file anyways?

The other question I had is that it seems like there's several duplicate checks being done. Is that because the scripts are different outside of the modified changes and we're really testing the other parts of the script? I'm fine if the answer is this is the way the test was written initially and we just need to update it to make them continue to work on newer versions of REDCap.

mmcev106 commented:
@kcmcg, did you see the notification for this one?  I figured I should ask since you just now reviewed #196 and it came in after this one.

mmcev106 commented:
@kcmcg, yes, the CSV was rewritten to support the other changes in this PR.  The `"csv" file containing the headings and rows` step requires exact matching, and huge `Lorem ipsum` rows would be pretty unwieldly to specify in feature files.  From what I can tell, there's no reason this CSV has to contain long values, so I rewrote it to use short values to allow me to make the features actually work.

I already added the following to my TODO list after this coming LTS version is certified:
- B.3.16.1400, B.3.16.1600, and B.3.16.1700 are all very similar, the last two being nearly identical.  Two of them are likely redundant.

The history of this project seems very messy with redundant tests and bad merges all over the place.  From my perspective getting tests passing is priority #1, with any cleanup tasks offering much less value than moving on to the next failing test at this time.

kcmcg reviewed (approved):
Thanks for the clarification! This all looks good to me.
